crates/cheatnet/src/cheatcodes/l1_handler_call.rs

```diff
@@ -0,0 +1,83 @@
+use crate::constants::build_block_context;
+use crate::state::DictStateReader;
+use crate::CheatnetState;
+use anyhow::Result;
+use blockifier::execution::{
+    errors::EntryPointExecutionError,
+    execution_utils::{felt_to_stark_felt, felts_as_str},
+};
+use blockifier::state::cached_state::CachedState;
+use blockifier::transaction::{
+    errors::TransactionExecutionError,
+    transactions::{ExecutableTransaction, L1HandlerTransaction},
+};
+use cairo_felt::Felt252;
+
+use starknet_api::core::{ContractAddress, EntryPointSelector};
+use starknet_api::hash::StarkFelt;
+use starknet_api::transaction::{Calldata, Fee, TransactionHash};
+
+use super::CheatcodeError;
+use crate::panic_data::try_extract_panic_data;
+
+impl CheatnetState {
+    pub fn l1_handler_call(
```

Comment:
Perhaps there is a more descriptive name for this cheatcode? Current naming might be misleading. I'm thinking about something like `mock_l1_call`, do you have any other ideas?

Reply:
@Radinyn IML `mock_l1_call` sounds more that we're trying to call the `L1`, or the `L1` is calling. Which is I think confusing too.

And also, it's not a `mock`, as we actually call the function.

The terminology `l1_handler` should appear. `call` sounds clear to me, and `l1_handler_mock` would be "we fake the call", which is not what we are doing.

WDYT?

Reply:
maybe `trigger_l1_handler`? mock sounds off to me as well